css/style.css

```diff
@@ -0,0 +1,112 @@
+body {
+  font-family: sans-serif;
+  font-size: 18px;
+  line-height: 1.5;
+  margin: 30px;
+  border-bottom: 5px solid #222222;
+  border-top: 5px solid #222222;
+  padding: 50px;
+  background: grey;
+  
+}
+
+
+h1 {
+  color: white;
+  font-family: 'Rock Salt', cursive;
+  text-transform: uppercase;
+  text-align: center;
+}
+
+
+
+
+
+h2 {
+  background-color: white;
+  font-family: 'Monoton', cursive;
+}
+
+
+
+
+
+
+h3 {
+  color: white;
+  font-size: 24px;
+
+}
+
+h4 {
+  font-family: 'Bungee Inline', cursive;
+
+}
+
+
+p {
+  font-family: 'Cherry Swash', cursive;
+
+
+}
+
+
+hr {
+  border: none;
+  border-top: 1px solid #222222;
+  margin: 30px 0;
+}
+
+
+
+strong {
+  color: white;
+}
+
+
+small {
+  color: yellow;
+}
+
+
+ul.awards {
+  text-align: left;
+  color: white;
+}
+
+
+li strong {
+  color: green;
+}
+
+
+
+img.witnesses {
```

Comment:
Combining tags and classes is not necessary and drives up the specificity without need.  Please read this article on specificity (https://css-tricks.com/specifics-on-css-specificity/) and BEM (https://csswizardry.com/2011/09/when-using-ids-can-be-a-pain-in-the-class/).  BEM is the most popular of the modular strategies to handle styling with classes (which is the industry best practice).  It is used by Google, AirBnB, etc, and is my go to way of organizing CSS.